## NOTE: dont put comments immediately after a key:string, comments will be interpreted as '\t'; also, usespace key instead of \TAB for ANY spacing!! (yaml-cpp is still buggy for parsing)

data:
  num_robots: 1

slam:
  num_particles: 100
  initial_w: 1.0
  # motion_noise: [0.1, 0.1, 0.1, 0.1, 0.1, 0.1]
  # foward_velocity, angular_velocity, theta
  motion_noise: [5, 5, 5, 5, 5, 5]
  
  measurement_noise: [0.0167, 0.000736589]